recipes/dlib/add_global_cxx_release.patch

```diff
@@ -0,0 +1,27 @@
+--- dlib/add_global_compiler_switch.cmake
++++ dlib/add_global_compiler_switch.cmake
+@@ -7,18 +7,18 @@
+ macro ( add_global_compiler_switch switch_name )
+    # If removing the switch would change the flags then it's already present
+    # and we don't need to do anything.
+-   string(REPLACE "${switch_name}" "" tempstr "${CMAKE_CXX_FLAGS}")
+-   if ("${CMAKE_CXX_FLAGS}" STREQUAL "${tempstr}" )
+-      set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${switch_name}" 
++   string(REPLACE "${switch_name}" "" tempstr "${CMAKE_CXX_FLAGS_RELEASE}")
```

Comment:
@jakirkham This is currently broken in Dlib master - but it's kind of a nasty hack that only makes it work for RELEASE mode. It definitely needs fixing upstream but I don't have the bandwidth to come up with a 'proper' fix at the moment.